dask/array/creation.py

```diff
@@ -173,13 +173,12 @@ def full_like(a, fill_value, dtype=None, chunks=None):
     )
 
 
-def linspace(start, stop, num=50, chunks=None, dtype=None):
+def linspace(start, stop, num=50, endpoint=True, retstep=False, chunks=None,
+             dtype=None):
```

Comment:
Guess this changes the argument order a little bit. Don't have any insightful thoughts on this ATM, but just checking to see if we should consider doing anything else or more here (warning, deprecation cycle, or nothing).

Reply:
Good catch @jakirkham, that was an oversight on my part. I'd be happy to change the order here so that it doesn't break any existing code. However, if we are okay changing the order, I'd be in favor of having the same order as `np.linspace` followed by `chunks`. I.e. 

```python 
def linspace(start, stop, num=50, endpoint=True, retstep=False, dtype=None, chunks=None):
```

Reply:
Yeah, I kind of like the order you have chosen as well. Moving `chunks` to the end seems fine to me (assuming we are ok with breaking the order). Let's see if others have any comments on this. Don't expect too much controversy as these are intended to be keyword arguments.